src/simpleocv.h

```diff
@@ -429,13 +429,15 @@ enum ImreadModes
 };
 
 NCNN_EXPORT Mat imread(const std::string& path, int flags = IMREAD_COLOR);
+NCNN_EXPORT Mat imread(const unsigned char* buffer, int len, int flags = IMREAD_COLOR);
 
 enum ImwriteFlags
 {
     IMWRITE_JPEG_QUALITY = 1
 };
 
 NCNN_EXPORT bool imwrite(const std::string& path, const Mat& m, const std::vector<int>& params = std::vector<int>());
+NCNN_EXPORT unsigned char* imwrite(int* len, const Mat& m);
 
```

Comment:
imdecode and imencode like how opencv highgui module says
simpleocv aims to emulate the exact opencv api, so 3rd software could use simpleocv instead of opencv without any code changes.